src/general/httpsonly/src.rego

```diff
@@ -1,8 +1,8 @@
 package k8shttpsonly
 
 violation[{"msg": msg}] {
-  input.review.kind.kind == "Ingress"
-  re_match("^(extensions|networking.k8s.io)$", input.review.kind.group)
+  input.review.object.kind == "Ingress"
```

Comment:
`input.review.kind` contains these values; there's no need for this change (likewise with the rego in the template)

Reply:
the test says otherwise:
```
  out = {
    "object": {
      "kind": "Ingress",
      "apiVersion": "extensions/v1beta1",
```
so is the test wrong and this is not a kubernetes object but something else ?

Reply:
it looks to me that all these policies are for kubernetes objects, and that therefore the old `input.review.kind.group` was wrong
if that's not the case, we should call that out "# policy for xyz system" and fix the tests

Reply:
@ctab I can merge the PR without fixing either the test or the source, so please lmk what solution you prefer

Reply:
@ctab I can't merge the PR without fixing either the test or the source, so please lmk what solution you prefer

Reply:
It does appear the existing test is wrong. Thanks for catching that. Could you add this to the review in the test?  (The object passed by Kubernetes contains these values so the source is valid.)

```
  "kind": {
    "group": "extensions",
    "version": "v1beta1",
    "kind": "Ingress"
  }
```

Reply:
ah yes, input.review.kind would be that, but the object has kind and apiVersion

Reply:
... I actually prefer the current code because it only uses the standard kubernetes objects that users can see via `get -o json` so it would be easier to debug/understand ?

Reply:
I can appreciate that. My preference would be to change the test since the scope of this PR is getting testing to work and the code is working as-is. But this is fine also.